containers/layouts/ViewLayoutRadios.js

```diff
@@ -26,17 +28,20 @@ const ViewLayoutRadios = ({ viewLayout, onChange, loading }) => {
         disabled: loading,
         name: 'viewLayout',
         label: c('Label to change view layout').t`Row`,
-        onChange: () => onChange(ROW),
+        async onChange() {
```

Comment:
Same, no need `async`
```suggestion
        onChange() {
```

Reply:
yeah, conflict with a previous version